arm_motors/src/main.cpp

```diff
@@ -239,10 +239,13 @@ void loop() {
         lastMotorStatus = millis();
 
         for (int i = 0; i < MOTOR_AMOUNT; i++) {
-            if (millis() - motorList[i]->status1.timestamp > 500)  // Don't send outdated data
-                continue;
-            COMMS_UART.printf("motorstatus,%d,%d,%d,%d\n", motorList[i]->getID(), int(motorList[i]->status1.motorTemperature * 10),
-                int(motorList[i]->status1.busVoltage * 10), int(motorList[i]->status1.outputCurrent * 10));
+            if (millis() - motorList[i]->status1.timestamp >= 500) {
```

Comment:
```suggestion
            if (motorList[i]->status1.timestamp - millis() < 500) {
```
this makes more sense to my brain but idk if it matters

this reads to me as "if it has been less than 500 ms then..."

Reply:
Keep this on the back burner

```c++
if (millis() < lastInstance + PERIOD)
```